src/app/clusters/time-synchronization-server/time-synchronization-server.cpp

```diff
@@ -473,6 +473,7 @@ void TimeSynchronizationServer::Init()
 void TimeSynchronizationServer::Shutdown()
 {
     PlatformMgr().RemoveEventHandler(OnPlatformEventWrapper, 0);
+    chip::Server::GetInstance().GetFabricTable().RemoveFabricDelegate(this);
```

Comment:
```suggestion
    Server::GetInstance().GetFabricTable().RemoveFabricDelegate(this);
```

Would be nice to remove the extra namespace prefixes in the Init method too...